apple-pie/styles/style.css

```
@@ -0,0 +1,95 @@
+.container{
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    font-family: 'Roboto', sans-serif;
+    height: 1000px;
+    width: 1520px;
```

vw es una medida relativa que al indicarle 100vw nos toma el ancho de la pantalla completa 😉 
```suggestion
    width: 100vw;
```

---

mas info https://devcode.la/tutoriales/unidades-vh-vw-css/